Migrating from Xerces-C++ 2.8.0 to Xerces-C++ 3.x

ianna opened the pull request:
Xerces-C++ 3.0.0 is a major release and includes a number of application-breaking interface changes compared to Xerces-C++ 2 series. The following sub-sections provide an overview of the public API changes between Xerces-C++ 2 series and this release.
- A number of DOM interfaces (DOMLSInput, DOMLSOuput, DOMLSParser, DOMLSSerializer, DOMConfiguration, etc.) were added as part of the the final DOM Level 3 specification conformance work
- Many public interfaces that used int/long types to represent memory-related sizes, counts, indexes, etc., have been modified to use the 64-bit safe XMLSize_t type instead
-  All APIs marked as deprecated in Xerces-C++ 2 series have been removed in this release. In particular deprecated DOM (depdom) as well as COM support have been removed.

Furthermore, a number of DOM interfaces (DOMBuilder, DOMWriter, DOMInputSource, etc.) were replaced as part of the the final DOM Level 3 specification conformance work.

Commit:
merge

Commit:
Merged refs/pull/13437/head from repository cms-sw

Commit:
remove no-longer used files

Commit:
fix

a-kapoor committed:
Merged fastsim-hittriplets-feb23 from repository lveldere

a-kapoor committed:
Primary Implementation of triplet Check

Commit:
Merged hitTriplet from repository ANSH0712

Commit:
fix some compilation issues

Commit:
refactor seedfinder selector

a-kapoor committed:
Merged fastsim-4anshul from repository lveldere

a-kapoor committed:
Removed a typo in a config

a-kapoor committed:
Implemented Full-Triplet-Check

a-kapoor committed:
Merged FastSim-Triplet from repository ANSH0712

a-kapoor committed:
Test

a-kapoor committed:
Merged Triplet-Check from repository ANSH0712

a-kapoor committed:
Split the triplet function in PixelTripletLargeTipGenerator so that iterations can directly use it for triplet check in FastSim

a-kapoor committed:
Implemedted triplet check for lowPtStep in FastSim Tracking and corrected a typo in the lowPtStep config

a-kapoor committed:
Merged Triplet-Check-In-FastSim from repository ANSH0712

a-kapoor committed:
Did some technical Changes in MultiHitGeneratorFromChi2 plugin in FullSim so that triplet check can be used in tobTecStep of FastSim Tracking

a-kapoor committed:
Added Triplet Check for seeds in TobTecStep and PixelLessStep in FastSim Tracking

Spudmeister committed:
Initial proposal for changes to HcalCalibDetId

jmmans committed:
Add a digi to hold information from the uMNio for laser and LED operations

jmmans committed:
Need more information and more methods here

jmmans committed:
Add code necessary to load/store the uMNqie flavor of HcalCalibDetId to text calibration files

Spudmeister committed:
Skeleton for uMNio unpacking

Spudmeister committed:
Added amc payload finding

Spudmeister committed:
Added unpacking of digi

Commit:
Merged Triplet-Check-In-FastSim_15Mar from repository ANSH0712

Commit:
do per event initialisation of MultiHitGeneratorFromChi2

grzanka committed:
CTPPS geometry and alignement

grzanka committed:
added missing TotemRPReco package

grzanka committed:
TotemRPReco package moved to right place

grzanka committed:
fixed AlignmentRecords

grzanka committed:
Removed Alignement algorithm code, will be added later

grzanka committed:
Obsolete files removed

grzanka committed:
RPTopology package refactored

grzanka committed:
Obsolete RPHepPDTWrapper removed

grzanka committed:
VeryForwadGeometryBuilder refactored

grzanka committed:
auto_ptr replaced with unique_ptr

a-kapoor committed:
Merged Triplet-Check-In-FastSim_Apr5 from repository lveldere

a-kapoor committed:
Implemented suggestions from experts after pull request

a-kapoor committed:
Removed some comments

a-kapoor committed:
Changes exception output

matt-komm committed:
Merged TripletCheckInFastSim from repository ANSH0712

matt-komm committed:
align coding style

matt-komm committed:
Merged for803p2 from repository davidlange6

Conflicts:
	FastSimulation/Tracking/plugins/TrajectorySeedProducer.cc

matt-komm committed:
algin code style

matt-komm committed:
improved selector interface

matt-komm committed:
Merged fastsim-81X-memleak from repository lveldere

Conflicts:
	FastSimulation/Tracking/plugins/TrajectorySeedProducer.cc

matt-komm committed:
fix segfault

grzankatest committed:
Removed DataFormats/TotemRPReco, already merged as CTPPSReco

pvmulder committed:
candidatebased ghosttrack tagger

pvmulder committed:
reverted back to original file + added the 2 ghostTrack cfi files

pvmulder committed:
missing configuration files

pvmulder committed:
ES needed too!

pvmulder committed:
bugfix in jetTools.py

pvmulder committed:
clean up of commented code, using CandidatePtr in GhostTrackComputer, not storing GT tagger in miniAOD for now

a-kapoor committed:
Merge branch 'improve-seedtree-interface' of https://github.com/matt-komm/cmssw into tripletseeding

a-kapoor committed:
Implemented Triplet and Doublet Selector in TrajectorySeedProducer

jmmans committed:
Support the CastorRadiationFacility calib detids

a-kapoor committed:
Removed couts and other technical changes

Commit:
Add the possibility of a FrontEnd map for HCAL

Commit:
Comment out one usage temporarily till DB is equipped

jmmans committed:
Fix issues raised by Slava

Commit:
Merged TripletSelectorInFastSim from repository ANSH0712

Commit:
fastsim: refactor selection in seedfinder

Commit:
add forgotten files

grzanka committed:
CTPPS Alignment decoupled

grzanka committed:
Commented out links to AlignmentGeometry

grzanka committed:
Lonely bracket commented out

kpedro88 committed:
fix more issues raised by Slava

grzanka committed:
removed problematic files
fixed couple of include guards

civanch committed:
fixed exceptions and printouts

a-kapoor committed:
Merged fastsim-tripletSeedSelection from repository lveldere

jmmans committed:
Fix swapped value1/value3

mulhearn committed:
clean-up L1T unpacker warnings

grzanka committed:
edm::EDAlayzer changed to edm::one::EDAnalyzer<>

Commit:
Fix deltaR calculation in widejet module

ianna committed:
Migrating from Xerces-C++ 2.8.0 to Xerces-C++ 3.x

ianna committed:
Merged xerces-3-port from repository ianna

ianna committed:
* Fix compilation errors

cmsbuild commented:
A new Pull Request was created by  @ianna (Ianna Osborne) for CMSSW_8_1_ROOT64_X.

It involves the following packages:

CalibCalorimetry/HcalTPGAlgos
Calibration/HcalCalibAlgos
CaloOnlineTools/HcalOnlineDb
CondTools/Ecal
CondTools/Hcal
DQM/CSCMonitorModule
DQM/HcalMonitorClient
DQM/SiPixelMonitorClient
DQM/SiStripMonitorClient
DetectorDescription/Parser
DetectorDescription/RegressionTest
GeneratorInterface/LHEInterface
IORawData/CaloPatterns
L1Trigger/RPCTrigger
PhysicsTools/MVATrainer
RecoBTag/XMLCalibration


@ghellwig, @covarell, @ianna, @cerminar, @rekovic, @vanbesien, @govoni, @civanch, @cmsbuild, @davidlange6, @Dr15Jones, @cvuosalo, @mdhildreth, @deguio, @slava77, @ggovi, @mmusich, @mulhearn, @perrozzi, @thuer, @dmitrijus, @franzoni can you please review it and eventually sign? Thanks.
@ghellwig, @apfeiffer1, @barvic, @mkirsano, @imarches, @threus, @acaudron, @tocheng, @argiro, @Martin-Grunewald, @agrohsje, @ferencek, @pvmulder, @venturia this is something you requested to watch as well.
@slava77, @smuzaffar you are the release manager for this.

cms-bot commands are list here cms-sw/cmssw#13028

ianna commented:
@iahmad-khan and @smuzaffar - FYI

ianna commented:
@smuzaffar - please, let me know if we decide to preserve backwards compatibility, I'd need to do the following modifications to the code:

```
#if _XERCES_VERSION <30100
#include "xercesc/dom/DOMWriter.hpp"
#else
 #include "xercesc/dom/DOMLSSerializer.hpp"
 #include "xercesc/dom/DOMLSOutput.hpp"
#endif
```

davidlange6 committed:
Merge pull request #14461 from kpedro88/uMNio-81X

Monitor CASTOR table data (81X)

cmsbuild committed:
Merge CMSSW_8_1_X into CMSSW_8_1_ROOT64_X.

Commit:
reco::MET --> reco::PFMET

smuzaffar commented:
cmsbuild, please test with cms-sw/cmsdist#2305

cmsbuild commented:
The tests are being triggered in jenkins.
Using externals from cms-sw/cmsdist#2305

cmsbuild commented:
-1
Tested at: UNKNOWN
 I was not able to find a release to test this PR. See the Jenkins logs for more details.

cmsbuild commented:
Comparison not run due to Build errors (RelVals and Igprof tests were also skipped)

cmsbuild commented:
-1

Tested at: e99d501fa5ecdcc7c2b8879aa25324f0effe1ffc


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/e27ac07f2b1392455fcf1e92abdabe500bc1d9ff
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14585/47/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14585/47/git-merge-result

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14585/47/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Building shared library tmp/slc6_amd64_gcc530/src/GeneratorInterface/PhotosInterface/src/GeneratorInterfacePhotosInterface/libGeneratorInterfacePhotosInterface.so
>> Building shared library tmp/slc6_amd64_gcc530/src/SimGeneral/HepPDTRecord/src/SimGeneralHepPDTRecord/libSimGeneralHepPDTRecord.so
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/cmsdist-cmssw-test-pr/CMSSW_8_1_DEVEL_X_2016-05-19-2300/src/GeneratorInterface/HiGenCommon/plugins/BetaBoostEvtVtxGenerator.cc 
tmp/slc6_amd64_gcc530/src/GeneratorInterface/LHEInterface/src/GeneratorInterfaceLHEInterface/LHEReader.o:(.data.rel.ro._ZTVN4lhef9LHEReader10XMLHandlerE[_ZTVN4lhef9LHEReader10XMLHandlerE]+0x40): undefined reference to `lhef::LHEReader::XMLHandler::ignorableWhitespace(unsigned short const*, unsigned long)'
tmp/slc6_amd64_gcc530/src/GeneratorInterface/LHEInterface/src/GeneratorInterfaceLHEInterface/LHEReader.o:(.data.rel.ro._ZTVN4lhef9LHEReader10XMLHandlerE[_ZTVN4lhef9LHEReader10XMLHandlerE]+0x188): undefined reference to `non-virtual thunk to lhef::LHEReader::XMLHandler::ignorableWhitespace(unsigned short const*, unsigned long)'
collect2: error: ld returned 1 exit status
gmake: *** [tmp/slc6_amd64_gcc530/src/GeneratorInterface/LHEInterface/src/GeneratorInterfaceLHEInterface/libGeneratorInterfaceLHEInterface.so] Error 1
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/cmsdist-cmssw-test-pr/CMSSW_8_1_DEVEL_X_2016-05-19-2300/src/GeneratorInterface/HiGenCommon/plugins/MixBoostEvtVtxGenerator.cc 
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/cmsdist-cmssw-test-pr/CMSSW_8_1_DEVEL_X_2016-05-19-2300/src/GeneratorInterface/HiGenCommon/plugins/HighMultiplicityGenFilter.cc 
Copying tmp/slc6_amd64_gcc530/src/GeneratorInterface/PhotosInterface/src/GeneratorInterfacePhotosInterface/libGeneratorInterfacePhotosInterface.so to productstore area:
Leaving library rule at GeneratorInterface/LHEInterface
</pre>


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/e27ac07f2b1392455fcf1e92abdabe500bc1d9ff
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14585/47/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14585/47/git-merge-result

grzanka committed:
remove DetectorDescription dependency
refactoring suggested by Ianna in https://github.com/CTPPS/cmssw/commit/b055ef9f9569427f81731c44032aa995f2005ff4

konec committed:
assigne eta=+-121 to CSC hits with keyWG above |eta|=1.3, then filter them
set q=4 to muons with |eta|=115,
remove layers 7,8,9 from list of layers with q-4
minor fixes to CSC keywire group assignment to eta code

davidlange6 committed:
Merge pull request #14456 from bsunanda/Run2-hcx82

bsunanda:Run2-hcx82 Add the possibility of a FrontEnd map for HCAL

davidlange6 committed:
Merge pull request #14591 from cms-l1t-offline/pr-unpacker-warning-cleanup

Clean-up L1T unpacker warnings

davidlange6 committed:
Merge pull request #14589 from duanders/fixpfscoutingmet81X

Use PFMETCollection in scouting PF producer (81X)

cmsbuild committed:
Merge CMSSW_8_1_X into CMSSW_8_1_ROOT64_X.

davidlange6 committed:
Merge pull request #14590 from duanders/widejetfix81X

Fix deltaR calculation in widejet module (81X)

cmsbuild committed:
Merge CMSSW_8_1_X into CMSSW_8_1_ROOT64_X.

konec committed:
slightly tune/modify condition of q=4 for |eta|==115.
Now it is done if in case |eta|==115 and hits are ONLY in 3 layers out of (7,8,9,17)

konec committed:
In case of |eta| 121 set its quality to 4.
It has no effect for MC shince hits with this quality are filtered out, but
is usefull for data-MC comparistion (if one remove the filter)

davidlange6 committed:
Merge pull request #14554 from ANSH0712/fastsim-tripletSeedSelectionbranch

Fastsim triplet seed selection branch

davidlange6 committed:
Merge pull request #14390 from pvmulder/GhostTrack-from-CMSSW_8_1_0_pre4

First version of the candidatebased GhostTrack tagger

davidlange6 committed:
Merge pull request #14029 from CTPPS/ctpps_geometry

CTPPS new reconstruction geometry related to TOTEM RP detectors

cmsbuild committed:
Merge CMSSW_8_1_X into CMSSW_8_1_ROOT64_X.

cmsbuild committed:
Merge pull request #14550 from civanch/more_sim_cleanup

Fixed Sim exceptions and printouts

cmsbuild committed:
Merge CMSSW_8_1_X into CMSSW_8_1_ROOT64_X.

ianna committed:
* Update Header

cmsbuild commented:
Pull request #14585 was updated. @ghellwig, @covarell, @ianna, @cerminar, @rekovic, @vanbesien, @govoni, @civanch, @cmsbuild, @davidlange6, @Dr15Jones, @cvuosalo, @mdhildreth, @deguio, @slava77, @ggovi, @mmusich, @mulhearn, @perrozzi, @thuer, @dmitrijus, @franzoni can you please check and sign again.

smuzaffar commented on GeneratorInterface/LHEInterface/src/LHEReader.cc:
```diff
@@ -127,8 +127,9 @@ class LHEReader::XMLHandler : public XMLDocument::Handler {
-	void characters(const XMLCh *const data, const unsigned int length) override;
-	void comment(const XMLCh *const data, const unsigned int length) override;
+        virtual void characters (const XMLCh *const chars, const XMLSize_t length) override;
+        virtual void ignorableWhitespace (const XMLCh *const chars, const XMLSize_t length) override;
```
@Ianna, looks like ignorableWhitespace definition is missing?

davidlange6 committed:
Merge pull request #14604 from cms-l1t-offline/pr-omtf-fix-81x

Update OMTF treatment of Muon at edge of OMTF acceptance

cmsbuild committed:
Merge CMSSW_8_1_X into CMSSW_8_1_ROOT64_X.

ianna committed:
Remove function declaration

cmsbuild commented:
Pull request #14585 was updated. @ghellwig, @covarell, @ianna, @cerminar, @rekovic, @vanbesien, @govoni, @civanch, @cmsbuild, @davidlange6, @Dr15Jones, @cvuosalo, @mdhildreth, @deguio, @slava77, @ggovi, @mmusich, @mulhearn, @perrozzi, @thuer, @dmitrijus, @franzoni can you please check and sign again.

ianna commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
-1
Tested at: UNKNOWN
 I was not able to find a release to test this PR. See the Jenkins logs for more details.

smuzaffar commented:
cmsbuild, please test with cms-sw/cmsdist#2305

it needs to be tested with cmsdist :-)

cmsbuild commented:
The tests are being triggered in jenkins.
Using externals from cms-sw/cmsdist#2305

cmsbuild commented:
-1
Tested at: UNKNOWN
 I was not able to find a release to test this PR. See the Jenkins logs for more details.

cmsbuild commented:
-1

Tested at: 5e533c6a7c00c27be8dc60f1951428ee3a2a1044

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14585/48/summary.html

I found follow errors while testing this PR

**Failed tests:  UnitTests  RelVals  AddOn**

* **Unit Tests**:

I found errors in the following unit tests: 
 
 ---> test testMVATrainer had ERRORS
---> test DetectorDescriptionParserTE had ERRORS
---> test reg_ddcore had ERRORS
---> test dumpDDCompactView had ERRORS
---> test dumpDDExpandedView had ERRORS
---> test testDDCompactView had ERRORS
---> test const_dump had ERRORS
---> test reg_ddparser had ERRORS
---> test DDCompareCPV had ERRORS
---> test DetectorDescriptionRegressionTestDDErrorReport had ERRORS
---> test runtestSimCalorimetryHGCalSimProducers had ERRORS


* **RelVals**:

When I ran the RelVals I found an error in the following worklfows:
4.22 step2
<pre>runTheMatrix-results/4.22_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC/step2_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC.log

</pre>140.53 step2
<pre>runTheMatrix-results/140.53_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI/step2_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI.log

</pre>8.0 step3
<pre>runTheMatrix-results/8.0_BeamHalo+BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS/step3_BeamHalo+BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS.log

</pre>1000.0 step2
<pre>runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/step2_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT.log

</pre>1001.0 step2
<pre>runTheMatrix-results/1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD1+ALCAHARVD2+ALCAHARVD3+ALCAHARVD4/step2_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD1+ALCAHARVD2+ALCAHARVD3+ALCAHARVD4.log

</pre>134.911 step3
<pre>runTheMatrix-results/134.911_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_2016+RECODR2_2016reHLT+HARVESTDR2/step3_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_2016+RECODR2_2016reHLT+HARVESTDR2.log

</pre>1330.0 step3
<pre>runTheMatrix-results/1330.0_ZMM_13+ZMM_13+DIGIUP15+RECOUP15+HARVESTUP15/step3_ZMM_13+ZMM_13+DIGIUP15+RECOUP15+HARVESTUP15.log

</pre>25.0 step3
<pre>runTheMatrix-results/25.0_TTbar+TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT/step3_TTbar+TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT.log

</pre>

* **AddOn**:

I found errors in the following addon tests:

cmsDriver.py TTbar_Tauola_13TeV_cfi -s GEN,SIM,DIGI,L1,DIGI2RAW --mc --scenario=pp -n 10 --conditions auto:run2_mc_GRun --relval 9000,50 --datatier "GEN-SIM-RAW" --eventcontent RAWSIM --customise=HLTrigger/Configuration/CustomConfigs.L1T --era Run2_2016 --magField 38T_PostLS1 --fileout file:RelVal_Raw_GRun_MC.root : FAILED - time: date Mon May 23 21:47:33 2016-date Mon May 23 21:43:51 2016 s - exit: 16640
cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi --conditions auto:run2_mc --fast  -n 100 --eventcontent AODSIM,DQM --relval 100000,1000 -s GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,DIGI2RAW,L1Reco,RECO,EI,HLT:@relval25ns,VALIDATION           --customise=HLTrigger/Configuration/CustomConfigs.L1THLT --datatier GEN-SIM-DIGI-RECO,DQMIO --beamspot NominalCollision2015 --era Run2_25ns --magField 38T_PostLS1 : FAILED - time: date Mon May 23 21:45:50 2016-date Mon May 23 21:43:52 2016 s - exit: 16640
cmsDriver.py TTbar_Tauola_13TeV_cfi -s GEN,SIM,DIGI,L1,DIGI2RAW --mc --scenario=pp -n 10 --conditions auto:run2_mc_PRef --relval 9000,50 --datatier "GEN-SIM-RAW" --eventcontent RAWSIM --customise=HLTrigger/Configuration/CustomConfigs.L1T --era Run2_2016 --magField 38T_PostLS1 --fileout file:RelVal_Raw_PRef_MC.root : FAILED - time: date Mon May 23 21:46:41 2016-date Mon May 23 21:43:53 2016 s - exit: 16640
cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi --conditions auto:run2_mc --fast  -n 100 --eventcontent AODSIM,DQM --relval 100000,1000 -s GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,DIGI2RAW,L1Reco,RECO,EI,HLT:@relval2016,VALIDATION  --customise=HLTrigger/Configuration/CustomConfigs.L1THLT --datatier GEN-SIM-DIGI-RECO,DQMIO --beamspot NominalCollision2015 --era Run2_2016 --magField 38T_PostLS1 : FAILED - time: date Mon May 23 21:52:27 2016-date Mon May 23 21:43:57 2016 s - exit: 16640
cmsDriver.py TTbar_Tauola_13TeV_cfi -s GEN,SIM,DIGI,L1,DIGI2RAW --mc --scenario=HeavyIons -n 10 --conditions auto:run2_mc_HIon --relval 9000,50 --datatier "GEN-SIM-RAW" --eventcontent RAWSIM --customise=HLTrigger/Configuration/CustomConfigs.L1T --era Run2_2016,Run2_HI --magField 38T_PostLS1 --fileout file:RelVal_Raw_HIon_MC.root : FAILED - time: date Mon May 23 21:49:14 2016-date Mon May 23 21:46:49 2016 s - exit: 16640
cmsDriver.py TTbar_Tauola_13TeV_cfi -s GEN,SIM,DIGI,L1,DIGI2RAW --mc --scenario=pp -n 10 --conditions auto:run2_mc_PIon --relval 9000,50 --datatier "GEN-SIM-RAW" --eventcontent RAWSIM --customise=HLTrigger/Configuration/CustomConfigs.L1T --era Run2_2016 --magField 38T_PostLS1 --fileout file:RelVal_Raw_PIon_MC.root : FAILED - time: date Mon May 23 21:55:34 2016-date Mon May 23 21:52:47 2016 s - exit: 16640

ianna committed:
Merged xerces-3-port from repository ianna

ianna committed:
* Fix run time test initialization
* Use unique_ptr instead of auto_ptr

cmsbuild commented:
Pull request #14585 was updated. @ghellwig, @covarell, @lveldere, @ianna, @cerminar, @Martin-Grunewald, @rekovic, @vanbesien, @govoni, @perrotta, @civanch, @cmsbuild, @fwyzard, @davidlange6, @smuzaffar, @Dr15Jones, @cvuosalo, @mdhildreth, @deguio, @slava77, @ggovi, @mmusich, @mulhearn, @ssekmen, @perrozzi, @thuer, @dmitrijus, @franzoni can you please check and sign again.
